### DISCUSSION TITLE
HUD ideas

### DISCUSSION THREAD

**Discussion by cloudchasingmeerkat** [cloudchasingmeerkat/laverna#25] [Ideas]
In #19 you've mentioned a main HUD with a money meter and a health meter. Unfortunately, Thief features no health meter boosting the immersion. How about the following HUD structure?

- Light meter 
- Item selection area
- Weapon selection area

Otherwise there are no visible elements in the HUD.

**Comment by Abb4** [comment]
@cloudchasingmeerkat could you maybe post an image of the thief ui and explain how it works. Or maybe just a video. For future reference.

**Comment by Abb4** [comment]
I think, a good UI fulfills the needs of the game. First it needs to be functional and then not bloated.

We definitely need to indicate light level. Light level is an indication of how easily a player can be discovered by an enemy (player visibility). It is therefore very important for a stealth game. Maybe we don't need a meter though. I remember an amazing visibility indicator in the game "the blackout club". It was the usual skyrim-eye detection indicator but very well polished and animated.

It indicated visibility, sound, enemy aggravation and even whether you are chased by a special, dangerous boss enemy, called "the shape". You can find all the indicators described here: https://theblackoutclub.fandom.com/wiki/How_To_Play:_HUD#Basics Or check out this video, note when combat begins: https://youtu.be/XYFCkrmHeZM?t=52

This eye would, of course, overlay indicators, so you could see noise sound waves and of course a full eye if you were in light. It was also animated, with animation transitioning between states (eye lids open and close).

@cloudchasingmeerkat could you tell me what the meter in thief does again? What do you think about the standard eye indicator?

**Comment by Abb4** [comment]
I actually would not mind a health indicator. The best health indicator, in my opinion, was in dishonored. Image for reference: https://i.ytimg.com/vi/q6Sn8k2vMv4/maxresdefault.jpg

The good thing about that indicator, is that when it was low it would start flashing, indicating that the player is close to death and needs to run. I find that really important and that needs to be communicated to the player.

One can overdo it though. Some games start flashing the entire screen red, if the player is close to dying, which I find unbearable.

But in dishonored if you ran away and hid, you health would regenerate on its own, after a short amount of time and all flashing would stop. You could still see, that your health was low, by looking at the indicator but it was not annoying anymore.

I think that that was the best compromise between indicating danger and keeping the ui clean.

@cloudchasingmeerkat what is the health system in thief like? How does it indicate low health?

**Comment by Abb4** [comment]
@cloudchasingmeerkat  As for the weapon selection area, do we really need it? Does thief have many weapons? Maybe make it fade away after selecting a weapon?
